app/(auth)/layout.tsx

```diff
@@ -15,4 +20,10 @@ const DashboardLayout = ({ children }: Readonly<{ children: React.ReactNode }>)
 	);
 };
 
-export default DashboardLayout;
+const LayoutWrapper: React.FC<{ children: React.ReactNode }> = ({ children }) => (
```

Comment:
If layout.tsx does not have ‘use client’ declared, declaring the provider might cause an error. I suggest moving this declaration to PageTitleProvider.tsx and exporting it, then importing it into layout.tsx. In the future, when the number of providers increases, we can consider creating a _providers folder to house all the providers.